frontend/src/lib/components/expander/Expander.svelte

```diff
@@ -66,47 +60,43 @@
-
-    - 'category' expander with base-300 as its base color for the title, and base-100
-     for the content. Text is black and bolded. Used to open a summary about some
-     area.
-
-    - 'question' expander with base-100 as its base color. Text is black and bolded.render
-     Used to get more info about an already answeared question.render
-
-    ### Properties
-
-    - 'title': Title used for the expander. This is also used as the aria-label for 
-     the checkbox on which the expander operates on.
-    - 'variant': The type for the expander.
-    - 'iconColor': The color for the icon. Default color is primary.
-    - 'iconPos': The position for the icon. Default is text, which means the icon will
-        be where the text ends.
-    - 'titleClass': Custom class string to add to the `<div>` containing the title.
-    - 'contentClass': Custom class string to add to the `<div>` containing the main content.
-
-    You should not try to use a variant and customize at the same time.
-
-    ### Usage
-
-    ```tsx
-    <Expander variant="read-more" title="Example title">
-    <p> Example content <p/>
-    </Expander>
-
-    <Expander title="Example title" iconColor="primary" customizeTitle=" bg-base-100 text-primary"
-    customizeContent=" bg-base-300 text-info font-bold">
-    <p> Example content <p/>
-    </Expander>
-    ```
-   -->
+  @component
+  A component for expanders that contain a title and some content. Use the
+  `variant` prop to specify the expander type.
+
+  - `read-more`: the default style of the expander. Used, for example, for getting
+    more information about a question.
+  - `question`: a more prominent style of the expander. Used in question listings
+    to display a question that can be expanded to reveal further information.
+  - `category`: the most prominent style of the expander. Used for collapsible
+    categories of items, such as questions.
+
+  ### Properties
+
+  - `title`: Title used for the expander. This is also used as the aria-label for 
+    the checkbox on which the expander operates on.
+  - `variant`: The type for the expander.
+  - `iconColor`: The color for the icon. Default color is primary.
+  - `iconPos`: The position for the icon. Default is text, which means the icon will
+      be where the text ends.
+  - `titleClass`: Custom class string to add to the `<div>` containing the title.
+  - `contentClass`: Custom class string to add to the `<div>` containing the main content.
+
+  You should not try to use a variant and customize at the same time.
+
+  ### Usage
+
+  ```tsx
+  <Expander title="Example title">
+    <p>Example content<p/>
+  </Expander>
+
+  <Expander title="Example title" variant="category"  iconColor="primary" 
+    titleClass="bg-base-100 text-primary" contentClass="bg-base-300 text-info font-bold">
+    <p>Example content<p/>
+  </Expander>
+  ```
+-->
 
 <div class={collapseClasses}>
   <input type="checkbox" aria-label="open ${title}" on:click={toggleExpanded} />
```

Comment:
What about the `aria-label` when the expander is supposed to close? Maybe we need to update the label dynamically.

Reply:
We should actually get rid of the label altogether and use this [pattern](https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/examples/disclosure-faq/). We had a discussion about that in the Candidate App PR that introduced `Expander`, but it was left unresolved.

I made an issue out of it #341